### PR TITLE
New package: slstatus-1.0

### PR DESCRIPTION
**Pull request by ribmo**
A package for [slstatus][1] to satisfy my own package request (https://github.com/voidlinux/void-packages/issues/4389).

[1]: https://github.com/drkh5h/slstatus

### CODE REVIEW

File: srcpkgs/slstatus/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+distfiles="https://git.nulltime.net/${pkgname}/snapshot/${pkgname}-${version}.tar.gz"
+makedepends="alsa-lib-devel libX11-devel"
+depends="alsa-lib"
+checksum=6b3510d6bc96b39a45cb2e2e02240ed7bc23eecb9b2fe79850dfe9f3b9ffff73
+
+do_build() {
+	cp ${FILESDIR}/config.h config.h
+	sed -i 's|^CFLAGS *=|override CFLAGS +=|g' config.mk
+	sed -i 's|^LDFLAGS *=|override LDFLAGS +=|g' config.mk
+	make
```
Comment (Duncaen) [inline srcpkgs/slstatus/template]: Rename this function to `pre_build` and let the gnu-makefile call `make`, this should fix all build issues.
The `do_install` function can be renamed to `post_install` to add the license.

**Comment by ribmo** [comment]
Damn, failing to build on armv*.

Anyone able to offer any help?

File: srcpkgs/slstatus/template (hunk 1 above)
Comment (ribmo) [inline srcpkgs/slstatus/template]: Thanks @Duncaen, that did the trick!

File: srcpkgs/slstatus/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+version=1.0
+revision=1
+build_style=gnu-makefile
+short_desc="Suckless status monitor for dwm"
+maintainer="Dom H. <dom@hxy.io>"
+license="GPL-3"
+homepage="https://git.nulltime.net/${pkgname}"
+distfiles="https://git.nulltime.net/${pkgname}/snapshot/${pkgname}-${version}.tar.gz"
+makedepends="alsa-lib-devel libX11-devel"
+depends="alsa-lib"
```
Comment (Duncaen) [inline srcpkgs/slstatus/template]: The `depends` is not necessary, `xbps-src` detects it itself.
See https://travis-ci.org/voidlinux/void-packages/jobs/141442314#L643

**Pull request merged**